crates/orchestrator/src/network.rs

```diff
@@ -375,13 +380,21 @@ impl<T: FileSystem> Network<T> {
         let chain_spec_raw_path = para_spec
             .build_chain_spec(&relay_chain_id, &self.ns, &scoped_fs)
             .await?;
-        scoped_fs.create_dir(para_spec.id.to_string()).await?;
+
+        // Para artifacts
+        let para_path_prefix = if let Some(custom_prefix) = custom_parchain_fs_prefix {
+            custom_prefix
+        } else {
+            para_spec.id.to_string()
+        };
```

Comment:
```suggestion
        let para_path_prefix = custom_parchain_fs_prefix.unwrap_or_else(|| para_spec.id.to_string());
```

Reply:
Not sure about this, I like `if let ..` but I'm open to change to `unwrap_or_else` if there is a good argument.